lldb/test/API/lang/swift/macro/TestSwiftMacro.py

```diff
@@ -34,19 +34,58 @@ def testDebugging(self):
         """Test Swift macros"""
         self.build(dictionary={'SWIFT_SOURCES': 'main.swift'})
         self.setupPluginServerForTesting()
+        main_spec = lldb.SBFileSpec('main.swift')
         target, process, thread, bkpt = lldbutil.run_to_source_breakpoint(
-            self, 'break here', lldb.SBFileSpec('main.swift'))
+            self, 'break here', main_spec
+        )
+
+        # We're testing line breakpoint setting here:
+        call_site_line = lldbtest.line_number("main.swift", "#no_return(a / b)")
+        call_site_bp = target.BreakpointCreateByLocation(main_spec, call_site_line)
 
         thread.StepOver()
         thread.StepInto()
+        
         # This is the expanded macro source, we should be able to step into it.
-        self.expect('reg read pc', substrs=[
-            '[inlined] freestanding macro expansion #1 of stringify in module a file main.swift line 5 column 11',
-            'stringify'
-        ])
+        # Don't check the actual line number so we are line independent
+        self.assertIn(
+            'freestanding macro expansion #1 of stringify in module a file main.swift line 5 column 11',
+            thread.frames[0].name,
+            "Stopped in stringify macro"
+        )
 
         self.expect('expression -- #stringify(1)', substrs=['0 = 1', '1 = "1"'])
 
+        # Step out should get us out of stringify, then in to the next macro:
+        thread.StepOut()
+        self.assertIn("a.testStringify", thread.frames[0].name, "Step out back to origin")
+        thread.StepInto()
+        self.assertIn(
+            "freestanding macro expansion #1 of no_return in module a file main.swift line 6 column 3",
```

Comment:
These demanglings may not be stable — but whoever changes them in the future can also update this test.

Reply:
Yeah, it was so handy that these names had the compiler's view of both the call site and the expansion order to confirm we hit the right one, and it serves as a check against our reading of the line table.  So I do think it's worth tracking changes in the demangling.